action.yml

```diff
@@ -2,14 +2,10 @@
 name: 'testground-composition'
 description: 'Run a Testground composition'
 inputs:
-  backend_addr:
+  backend_endpoint:
```

Comment:
If we wanted to make this backwards compatible, then we could use `backend_endpoint` if it's defined and fall back to `backend_proto` + `backend_addr` otherwise.

I don't feel strongly about making it backwards compatible though.